File: csharpServer/model/Seat.cs
namespace model;

/// <summary>
/// Simple value object representing a seat number inside a coach (1-18).
/// </summary>
public class Seat
{
    public int SeatNumber { get; }
    public string ClientName { get; }

    public Seat(int seatNumber, string clientName)
    {
        SeatNumber = seatNumber;
        ClientName = string.IsNullOrEmpty(clientName) ? "-" : clientName;
    }
    
    public override string ToString()
    {
        return $"Seat {SeatNumber} reserved by {ClientName}";
    }
}

File: javaClient/model/src/main/java/ro/transportcompany/Seat.java
package ro.transportcompany;

public class Seat {
    private final int seatNumber;
    private final String clientName;

    public Seat(int seatNumber, String clientName) {
        this.seatNumber = seatNumber;
        this.clientName = clientName;
    }

    public int getSeatNumber() {
        return seatNumber;
    }

    public String getClientName() {
        return clientName == null ? "-" : clientName;
    }
}